File: packages/cli/test/fixtures/ranking-corpora/java/src/main/java/com/example/ranking/handlers/DeleteHandler.java
package com.example.ranking.handlers;

import com.example.ranking.util.Logger;

/** Removes a user by id. */
public class DeleteHandler {
    public void deleteUser(int id) {
        if (id <= 0) {
            Logger.logError("deleteUser", new IllegalArgumentException("id must be positive"));
            return;
        }
        Logger.logInfo("deleted user");
    }
}

File: packages/cli/test/fixtures/ranking-corpora/kotlin/src/main/kotlin/com/example/ranking/handlers/DeleteHandler.kt
package com.example.ranking.handlers

import com.example.ranking.util.Logger

/** Removes a user by id. */
class DeleteHandler {
    fun deleteUser(id: Int) {
        if (id <= 0) {
            Logger.logError("deleteUser", IllegalArgumentException("id must be positive"))
            return
        }
        Logger.logInfo("deleted user")
    }
}
